### DISCUSSION TITLE
How to compile modify this for an x86_64 / amd64 linux platform?

### DISCUSSION THREAD

**Discussion by rwaiboer** [petrvlcek/bisecur2mqtt#11] [Q&A] [answered]
Hi,

Trying to run this docker on an x86_64 platform, but get the following error:
"exec /usr/bin/entry.sh: exec format error"
After some investigation I see that the docker is created for a raspberry pi (e.g. Dockerfile refers to "FROM balenalib/raspberry-pi-openjdk:8-stretch-run" ).
So tried to compile for x86_64, however the instructions for compiling for a different platform are at this moment beyond my knowledge. I would like to help develop this further, but need a bit of help as I have some programming experience, but have no docker development experience.

Regards,

Rob

**Comment by rwaiboer** [comment, marked as answer]
Solved by installing gemu:
sudo apt-get install -y qemu qemu-user-static

**Comment by GD-Dal** [comment]
Nice. But can you get this program to work at all? I cannot.
Did you do something else? Configuration, etc
